test/e2e/manifest_test.go

```diff
@@ -49,6 +49,12 @@ var _ = Describe("Podman manifest", func() {
 		Expect(session.ExitCode()).To(Equal(0))
 	})
 
+	It("podman manifest inspect", func() {
+		session := podmanTest.PodmanNoCache([]string{"manifest", "inspect", "docker://library/busybox"})
```

Comment:
Is this an image we have *cached*? The bug here is that we are failing to run `podman manifest list` on local images we have already pulled, but who manifests were not created by Podman?

Reply:
> The bug here is that we are failing to run `podman manifest list` on local images we have already pulled, but who manifests were not created by Podman?

yes, this needs to be fixed. And the steps to reproduce the issue is trying to inspect the manifest on a registry server. So manifest inspect is supposed to work no matter the image is in local storage or not.